ios/mapbox_maps_flutter.podspec

```diff
@@ -16,7 +16,7 @@ Pod::Spec.new do |s|
   s.source_files = 'Classes/**/*'
   s.dependency 'Flutter'
   s.platform = :ios, '11.0'
-  s.dependency 'MapboxMaps', '~> 10.10.0'
+  s.dependency 'MapboxMaps', '~> 10.13.1'
```

Comment:
10.13.1 here while 10.13.0 for Android - AFAIR it's OK but double-check please

Reply:
Yes since https://github.com/mapbox/mapbox-maps-android/releases/tag/v10.13.0 is the latest Android and iOS has https://github.com/mapbox/mapbox-maps-ios/releases/tag/v10.13.1